lib/rules/display-name.js

```diff
@@ -70,6 +77,36 @@ module.exports = {
       return node.type === 'CallExpression' && argumentIsCallExpression && utils.isPragmaComponentWrapper(node);
     }
 
+    /**
+     * Checks if the node is a React.createContext call
+     * @param {ASTNode} node - The AST node being checked.
+     * @returns {Boolean} - True if node is a React.createContext call object literal, false if not.
+     */
+    function isCreateContext(node) {
+      if (!node.init) {
+        return false;
```

Comment:
this line is what we'd need to hit - when would `node` have a falsy `init` value?

Reply:
Initially I put it in here as a guard without much thought. This comment got me thinking that contexts declared like this were not covered by the rule

```

Reply:
Initially I put this line in as guard without much thought. Thinking about it, `node.init` can only be falsy with a variable declaration without an initial value eg `let Hello;`. This got me thinking about the following: The rule would not detect contexts in assignments like this

```js
import { createContext } from 'react';

var Hello = {};
Hello = React.createContext(); // or createContext()
```

While unusual, I added a check for this too along with tests for it.

Reply:
Awesome, thanks!